api/endpoint.go

```diff
@@ -89,24 +97,56 @@ func (e *Endpoint) makeDataRequest(
 	request DataRequest,
 ) {
 	prepareRequestLogging(ctx, request)
-	conn, err := e.MakeVdsConnection(request.credentials())
-	if abortOnError(ctx, err) {
+
+	vdsUrls, sasTokens, binaryOperatorString := request.credentials()
+
+	binaryOperator, err := core.GetBinaryOperator(binaryOperatorString)
+	if err != nil {
```

Comment:
`if abortOnError(ctx, err) {`
?

Now if I send invalid data I get 200 and empty response. We might need a test.